plugins/dbnd-docker/src/dbnd_docker/kubernetes/kube_dbnd_client.py

```diff
@@ -599,8 +604,12 @@ def handle_pod_retry(
                 )
                 return False
         elif reason == PodRetryReason.err_image_pull:
-            retry_count = self.kube_config.retry_on_image_pull_error_count
-            retry_delay = self.kube_config.pod_retry_delay
+            retry_count = self.kube_config.pod_err_cfg["exit_code"]["image_pull_error"][
```

Comment:
Should be changed to `get_retry_count` and `get_retry_delay`

Reply:
yes, my bad